rsts/deployment/plugin_setup/k8s/index.rst

```diff
@@ -1,57 +1,414 @@
 .. _deployment-plugin-setup-k8s:
 
-################
-K8s Plugin Setup
-################
 
+K8s Operator backend Plugin Setup
+-----------------------------------------
 
-.. panels::
-    :header: text-center
+This guide gives an overview of how to set up the K8s Operator backend plugin in your Flyte deployment.
 
+1. Add the flyte chart repo to helm
 
-    .. link-button:: deployment-plugin-setup-k8s-mpi-operator
-       :type: ref
-       :text: MPI Operator
-       :classes: btn-block stretched-link
-    ^^^^^^^^^^^^
-    Guide to setting up the Kubeflow MPI Operator Plugin.
+  .. code-block::
 
-    ---
+     helm repo add flyteorg https://flyteorg.github.io/flyte
```

Comment:
<img width="732" alt="image" src="https://user-images.githubusercontent.com/27777173/145152082-1b0d4446-8cf2-4602-9072-31e94d63b376.png">

Can we somehow remove the line?

Reply:
Yes it's tabbing issue, I fixed it